net.zgcoder.skycheckers.json

```diff
@@ -54,9 +54,13 @@
 			"sources": [
 				{
 					"type": "archive",
-					"url": "https://github.com/zorgiepoo/Sky-Checkers/releases/download/1.2/sc-snap-pak-repo.tgz",
-					"sha256": "ff0a0e7a608e52a3b8b1d8d7fadfc05a953b34142e5fc5af943f2e1b1bd2057d"
-				}
+					"url": "https://github.com/zorgiepoo/Sky-Checkers/archive/refs/tags/v1.3.tar.gz",
+					"sha256": "b9ed99216def4e4b5d3609f016f9be2502b0734db4d8548af62ffb2517668d68"
```

Comment:
I think I need to create a `sc-snap-pak-repo.tgz` for 1.3 otherwise full git repo is overkill.

Reply:
Yea sure, if you add it in a release I can drop the release patch.

Reply:
Changed to sc-snap-pak-repo.tgz for 1.3